README.md

```diff
@@ -68,34 +71,38 @@ This repository's contents is divided across four primary sections:
 - `/packages` contains the source for our public packages
 - `/scripts` contains scripts used for building and publishing
 - `/assets` contains icons and translations relied on by the app
+- `/docs` contains the content for our docs site at [tldraw.dev](https://tldraw.dev)
 
 ### Applications
 
 - `examples`: our local development / examples project
-- `docs`: our docs site at [tldraw.dev](https://tldraw.dev)
 - `vscode`: our [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=tldraw-org.tldraw-vscode)
 
 ### Packages
 
+- `assets`: a library for working with tldraw's fonts and translations
 - `editor`: the tldraw editor
-- `ui`: the editor's user interface
-- `tldraw`: the main tldraw package containing both the editor and the UI
+- `file-format`: a library for working with tldraw's `.tldr` file format
+- `indices`: a library for working with tldraw's indices
+- `polyfills`: a collection of polyfills used by tldraw
 - `primitives`: low-level primitives for working with vectors and geometry
+- `state`: a signals library, also known as signia
+- `store`: an in-memory reactive database
+- `tldraw`: the main tldraw package containing both the editor and the UI
 - `tlschema`: shape definitions and migrations
-- `tlsync-client`: a library for (locally) syncronizing editor instances
-- `tlstore`: an in-memory reactive database
-- `tlvalidate`: a validation library used for run-time validation
-- `tlassets`: a library for working with tldraw's fonts and translations
-- `file-format`: a library for working with tldraw's `.tldr` file format
+- `ui`: the editor's user interface
 - `utils`: low-level data utilities shared by other libraries
+- `validate`: a validation library used for run-time validation
 
 ## Community
 
 Have questions, comments or feedback? [Join our discord](https://discord.gg/rhsyWMUJxd) or [start a discussion](https://github.com/tldraw/tldraw/discussions/new).
 
 ## Distributions
 
-You can find tldraw on npm [here](https://www.npmjs.com/package/@tldraw/tldraw).
+You can find tldraw on npm [here](https://www.npmjs.com/package/@tldraw/tldraw?activeTab=versions).
+
+At the moment the tldraw package is in alpha. We also ship a canary version which is always up to date with the main branch of this repo.
```

Comment:
Updated to signpost that we're on alpha still. Bit wordy in this context - could maybe do with an edit.